src/Pages/Wallets.razor

```diff
@@ -492,11 +457,54 @@
                             </FieldBody>
                         </Field>
                     </Column>
-
-
+                    <Column ColumnSize="ColumnSize.Is6" @bind-Value="@_amountToTransfer">
+                        <Field>
+                            <FieldLabel><strong>From</strong> (this wallet)</FieldLabel>
+                            <FieldBody>
+                                <div class="flex-container">
+                                    <p class="source-wallet-name">@_sourceWalletName</p>
+                                    <div class="arrow-icon-container">
+                                        <Icon Name="IconName.ArrowRight" class="arrow-icon" />
+                                    </div>
+                                </div>
+                            </FieldBody>
+                            <FieldHelp class="field-help">
+                                Current balance: @_sourceWalletBalance
+                            </FieldHelp>
+                            <FieldHelp>
+                                Balance after transfer: @CalculateAmountWalletSourceAfterTransaction():f8 BTC (@Math.Round(PriceConversionService.BtcToUsdConversion(_sourceBalanceAfterTransaction, _btcPrice), 2) USD)
```

Comment:
This happens when you transfer all your funds or your balance in the wallet is 0

<img width="791" alt="Screenshot 2023-10-11 at 16 47 08" src="https://github.com/Elenpay/NodeGuard/assets/39995243/adfaf95e-e7e9-44dd-a20e-4c8fffbb91b6">

Reply:
Also, if you make a withdrawal and open the modal again, it has the same info as the previous transaction, it should be cleared